### DISCUSSION TITLE
Chrome License for Firefox

### DISCUSSION THREAD

**Discussion by Maribloemer** [aclap-dev/video-downloadhelper#451] [License & Purchase requests]
### Your question

I bought a license for Google Chrome, but it doesn't work in that browser. Do not download the videos.

I used it on the Firerox for 2 days and it worked, but now it's giving an error. How do I change my Chrome license to Firefox?

### Addon & CoApp version and details

.

**Comment by jcfrog** [comment]
Hi,
You can't switch licenses.
Chrome and Firefox premium are separate products with different prices and features.
If your purchase was made less than one month ago we can refund your Chrome premium so that you can purchase Firefox premium if you want to.
Please send us an email at licenses@downloadhelper.net

**Reply by huutrieu102** [reply]
Hi, I have the same problem, can you help me?

**Reply by jcfrog** [reply]
Please send us an email at [licenses@downloadhelper.net](mailto:licenses@downloadhelper.net)
